chart/tests/test_flower.py

```diff
@@ -352,7 +352,7 @@ def test_defaults(self):
         assert [{"namespaceSelector": {"matchLabels": {"release": "myrelease"}}}] == jmespath.search(
             "spec.ingress[0].from", docs[0]
         )
-        assert [{"port": "flower-ui"}] == jmespath.search("spec.ingress[0].ports", docs[0])
+        assert [{"port": 5555}] == jmespath.search("spec.ingress[0].ports", docs[0])
```

Comment:
This look good, but we should move the named-port coverage into the following test, `test_ports_overrides`.